google/cloud/firestore_v1/document.py

```diff
@@ -76,7 +76,7 @@ def create(self, document_data) -> Any:
         write_results = batch.commit()
         return _first_write_result(write_results)
 
-    def set(self, document_data, merge=False) -> Any:
+    def set(self, document_data, merge=False) -> Coroutine:
```

Comment:
This method returns a single `google.cloud.proto.firestore.v1.write.WriteResult` instance.
```suggestion
    def set(self, document_data, merge=False) -> write.WriteResult:
```

Reply:
I have already tried that but `pytype` throws an error like `Expected: google.cloud.firestore_v1.types.write.WriteResult
  Actually returned: coroutine`
Also found used `coroutine` in async_document.py file
https://github.com/googleapis/python-firestore/blob/2021f38bb6f016c13bc43d59730c77b57ae5c352/google/cloud/firestore_v1/async_document.py#L78